beetmoverscript/task.py

```diff
@@ -134,6 +138,27 @@ def get_upstream_artifacts(context):
     return artifacts
 
 
+def get_release_props(context, platform_mapping=STAGE_PLATFORM_MAP):
+    """determined via parsing the Nightly build job's balrog_props.json and
+    expanded the properties with props beetmover knows about."""
+    payload_properties = context.task.get('payload', {}).get('release_properties', None)
+    initial_release_props_file = None
+
+    if payload_properties:
+        props = payload_properties
+        log.debug("Loading release_props from task's payload: {}".format(props))
+    else:
+        initial_release_props_file = get_initial_release_props_file(context)
+        props = utils.load_json(initial_release_props_file)['properties']
+        log.warn(
+            'Deprecated behavior! This will be gone after Firefox 59 reaches release. Loading release_props from "{}": {}'
```

Comment:
Tiny nit: maybe split here the `Loading ...` part on a newline to cope with PEP8.

Reply:
Why not. FWIW, we had a discussion about line length a bit less than a year ago. The max line length used within releng projects is 160. For reference: https://github.com/mozilla-releng/services/pull/282

Reply:
👍